AirCasting/CreateSessionViews/SelectDeviceView.swift

```diff
@@ -93,7 +93,19 @@ struct SelectDeviceView: View {
                     isTurnOnBluetoothLinkActive = true
                 }
             } else if selected == 2 {
-                isMicLinkActive = true
+                if microphoneManager.recordPermissionGranted() {
```

Comment:
I created this function instead of checking here if audiosession.recordPermission == .granted. I thought maybe it would be better if microphoneManager was the one to know what the recordPermission should be equal to if permission is granted. But let me know if you prefer your solution with var recordPermission